ignite/metrics/recall.py

```diff
@@ -171,6 +167,9 @@ def __init__(
         is_multilabel: bool = False,
         device: Union[str, torch.device] = torch.device("cpu"),
     ):
+
+        if average not in [False, "micro", "samples"]:
```

Comment:
Why Recall can't be weighted ?

Reply:
Weighted recall is equivalent with accuracy. For example, in binary case we have:
![image](https://user-images.githubusercontent.com/22097587/168660324-6daf34ef-d604-463e-b9e4-e9f56ca3fad0.png)

in which 
![image](https://user-images.githubusercontent.com/22097587/168660825-5009a2e6-6edd-47c7-b032-3ae949c37390.png)

Reply:
This is stated [here](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html) as well.

Reply:
I think it is not a problem that Weighted Recall == Accuracy. IMO we could support it as scikit-learn does.

Reply:
So, supported options for `Precision` and `Recall` would be equal.